src/main/java/io/vertx/rabbitmq/RabbitMQueue.java

```diff
@@ -0,0 +1,86 @@
+package io.vertx.rabbitmq;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.streams.ReadStream;
+
+/**
+ * A stream of messages from a rabbitmq queue.
+ */
+@VertxGen
+public interface RabbitMQueue extends ReadStream<JsonObject> {
```

Comment:
I think it should be named `RabbitMQConsumer` because it's the interface to consumer a queue.

Reply:
Agree, done